akka-persistence-testkit/src/main/scala/akka/persistence/testkit/query/internal/EventsByPersistenceIdStage.scala

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright (C) 2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.persistence.testkit.query.internal
+import akka.actor.ActorRef
+import akka.annotation.InternalApi
+import akka.persistence.query.{ EventEnvelope, Sequence }
+import akka.persistence.testkit.{ EventStorage, PersistenceTestKitPlugin }
+import akka.persistence.testkit.EventStorage.{ NoMetadata, WithMetadata }
+import akka.stream.{ Attributes, Outlet, SourceShape }
+import akka.stream.stage.{ GraphStage, GraphStageLogic, GraphStageLogicWithLogging, OutHandler }
+import com.github.ghik.silencer.silent
+
+/**
+ * INTERNAL API
+ */
+@InternalApi
+@silent
```

Comment:
that's very silent

Reply:
I think a little more noise is a good thing